binding/python/util.py

```diff
@@ -125,13 +126,18 @@ def _pv_keyword_files_subdir():
 
     raise NotImplementedError('Unsupported platform')
 
-
-def pv_keyword_paths(relative):
+def pv_keyword_paths(relative, lang):
+    keyword_files_root = _append_by_lang('resources/keyword_files', lang)
     keyword_files_dir = \
-        os.path.join(os.path.dirname(__file__), relative, 'resources/keyword_files', _pv_keyword_files_subdir())
+        os.path.join(os.path.dirname(__file__), relative, keyword_files_root, _pv_keyword_files_subdir())
 
     res = dict()
     for x in os.listdir(keyword_files_dir):
         res[x.rsplit('_')[0]] = os.path.join(keyword_files_dir, x)
 
     return res
+
+def _append_by_lang(dir, lang):
```

Comment:
you should remove this since it won't be used in the binding - if you find you need a util for just the tests, create a test_utils file